tests/functional/test_change_email_address.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from flask import url_for
+
+q_email = 'some.one@digital.cabinet-office.gov.uk'
+
+email_depts = [('some.one@cabinetoffice.gov.uk',
+                'Cabinet Office')]
+
+
+class When_on_change_email(object):
+
+    def it_shows_the_email_address_query_string_in_the_textbox(
```

Comment:
This actually highlights a problem with the view code - it should not accept an email address from the URL query params, but should use the email address in the session. This should be fixed (perhaps a new ticket).

This test name reveals too much knowledge of the implementation - it should be something like `it_shows_the_user_supplied_email_in_the_textbox`.